### PR TITLE
Add test cases

### PR DESCRIPTION
**Pull request by kenzantonius**

### CODE REVIEW

File: src/test/java/expresslibrary/testutil/TypicalIndexes.java [hunk 1]
```diff
@@ -9,4 +9,7 @@ public class TypicalIndexes {
     public static final Index INDEX_FIRST_PERSON = Index.fromOneBased(1);
```
Comment (Jarrett0203) [inline src/test/java/expresslibrary/testutil/TypicalIndexes.java:9]: Maybe you can rename this to just `INDEX_FIRST` and not create `INDEX_FIRST_BOOK`? Its the same thing anyway

File: src/test/java/expresslibrary/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -39,13 +39,20 @@ public class CommandTestUtil {
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
 
-    public static final String VALID_TITLE_ANIMAL = "Animal Farm";
-    public static final String VALID_TITLE_HARRY = "Harry Potter";
-    public static final String VALID_AUTHOR_ALEX = "Alexandra Harris";
```
Comment (Jarrett0203) [inline src/test/java/expresslibrary/logic/commands/CommandTestUtil.java:44]: I think you might have accidentally deleted this? That might be why the test cases are failing.

**Review by kynhan — DISMISSED** [review]
LGTM

File: src/test/java/expresslibrary/logic/commands/CommandTestUtil.java [hunk 3]
```diff
@@ -36,13 +37,22 @@ public class CommandTestUtil {
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
 
+    public static final String VALID_AUTHOR_ROWLING = "JK Rowling";
+    public static final LocalDate VALID_BORROW_DATE = LocalDate.parse("30/03/2023");
```
Comment (Jarrett0203) [inline src/test/java/expresslibrary/logic/commands/CommandTestUtil.java:41]:
```suggestion
    public static final LocalDate VALID_BORROW_DATE = DateUtil.parseDate("30/03/2023");
```

File: src/test/java/expresslibrary/logic/commands/CommandTestUtil.java [hunk 4]
```diff
@@ -36,13 +37,22 @@ public class CommandTestUtil {
...
+    public static final Person VALID_BORROWER_ALICE = TypicalPersons.ALICE;
+    public static final LocalDate VALID_DUE_DATE = LocalDate.parse("06/04/2023");
```
Comment (Jarrett0203) [inline src/test/java/expresslibrary/logic/commands/CommandTestUtil.java:43]:
```suggestion
    public static final LocalDate VALID_DUE_DATE = DateUtil.parseDate("06/04/2023");
```

File: src/test/java/expresslibrary/logic/commands/CommandTestUtil.java [hunk 5]
```diff
@@ -140,10 +162,9 @@ public static void showBookAtIndex(Model model, Index targetIndex) {
         assertTrue(targetIndex.getZeroBased() < model.getFilteredBookList().size());
 
         Book book = model.getFilteredBookList().get(targetIndex.getZeroBased());
-        final String[] splitName = book.getTitle().title.split("\\s+");
-        model.updateFilteredBookList(new TitleContainsKeywordsPredicate(Arrays.asList(splitName[3])));
+        final String[] splitTitle = book.getTitle().title.split("\\s+");
+        model.updateFilteredBookList(new TitleContainsKeywordsPredicate(Arrays.asList(splitTitle[0])));
```
Comment (Jarrett0203) [inline src/test/java/expresslibrary/logic/commands/CommandTestUtil.java:166]: Change this back to 3, it's for `ListBookCommandTest`.
```suggestion
        model.updateFilteredBookList(new TitleContainsKeywordsPredicate(Arrays.asList(splitTitle[3])));
```

**Pull request closed without merging**